pkg/config/templates/rke2-92-harvester-kube-audit-policy.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: audit.k8s.io/v1
+kind: Policy
+rules:
+- level: Metadata
```

Comment:
log all requests at metadata level is too much, need to define mroe appropriate rules for the Harvester resources, e.g., only tracks pod, VMs, pvc(volumes), images and  networks and harvester related CRD might be enough for now.

Reply:
With the test of only record `create, delete, patch` action, audit log is reduced 

We can try to exclude more resources, but it takes time to test per rule manually, leave it blank now.

```
harv41:~ # ls -alth /var/lib/rancher/rke2/server/logs/audit.log
-rw------- 1 root root 1.3M Sep 15 21:51 /var/lib/rancher/rke2/server/logs/audit.log

harv41:~ # ls -alth /var/lib/rancher/rke2/server/logs/audit.log
-rw------- 1 root root 2.2M Sep 15 22:03 /var/lib/rancher/rke2/server/logs/audit.log
```